testing/terraform/eks/main.tf

```diff
@@ -16,9 +16,7 @@ terraform {
   }
 }
 
-provider "aws" {
-  region = var.aws_region
-}
+provider "aws" {}
```

Comment:
Put this back and instead add `-var="aws_region=${{ env.AWS_DEFAULT_REGION }}" \` in the workflow here: https://github.com/aws-observability/aws-otel-java-instrumentation/blob/main/.github/workflows/appsignals-e2e-eks-test.yml#L262

This should work and still maintain the expected behaviour that if someone doesn't explicitly pass in the region it fails.